packages/math/init.lua

```diff
@@ -67,16 +68,28 @@ function package:registerCommands ()
     xpcall(function()
       mbox = self:ConvertMathML(content, mbox)
     end, function(err) print(err); print(debug.traceback()) end)
-    self:handleMath(mbox, mode)
+    self:handleMath(mbox, mode, nil)
```

Comment:
I don't use the MathML syntax much (vs. TeXlike math syntax), but display math equations could be numbered there too, no?

Reply:
I have never used it, but yes, they could be also numbered (I need to test it).